### PR TITLE
chore(utils): replace all the unneeded spec usage for `westend-(dev|local)` runtime

### PR DESCRIPTION
**Pull request by EclesioMeloJunior**
## Changes

- Replace the usage of all `GetGssmrGenesis*` utils functions for `GetWestendDevGenesisPath()` in order to remove unneeded spec usage

## Tests

N/A

## Issues

- Ref: https://github.com/ChainSafe/gossamer/issues/2895

### CODE REVIEW

File: lib/utils/utils.go [hunk 1]
```diff
@@ -148,41 +148,14 @@ func KeystoreFilepaths(basepath string) ([]string, error) {
...
-	path, err := GetGssmrGenesisRawPath()
-	require.NoError(t, err)
-	return path
-}
-
-// GetGssmrGenesisRawPath gets the gssmr raw genesis path
+// GetGssmrGenesisPath gets the gssmr genesis path
 // and returns an error if it cannot find it.
-func GetGssmrGenesisRawPath() (path string, err error) {
+func GetGssmrGenesisPath() (path string, err error) {
```
Comment (qdm12) [inline lib/utils/utils.go:153]: Noob question, could we get rid of that one?

File: tests/rpc/rpc_01-system_test.go [hunk 2]
```diff
@@ -77,7 +80,7 @@ func TestSystemRPC(t *testing.T) { //nolint:tparallel
 				return false, err // error and stop retrying
 			}
 
-			ok = healthResponse.Peers == numberOfNodes-1 && !healthResponse.IsSyncing
+			ok = healthResponse.Peers == numberOfNodes-1
```
Comment (qdm12) [inline tests/rpc/rpc_01-system_test.go:83]: Are you sure about this change? I'm looking why into why it's flaky,

File: tests/rpc/rpc_03-chain_test.go [hunk 3]
```diff
@@ -103,11 +103,11 @@ func TestChainRPC(t *testing.T) {
 		assert.Regexp(t, regexBytesHex, digestLog)
 	}
 	block.Block.Header.Digest.Logs = nil
-	assert.Len(t, block.Block.Body, 1)
+	assert.GreaterOrEqual(t, len(block.Block.Body), 1)
```
Comment (qdm12) [inline tests/rpc/rpc_03-chain_test.go:106]: nit

```suggestion
	assert.NotEmpty(t, block.Block.Body)
```

File: tests/rpc/rpc_03-chain_test.go [hunk 4]
```diff
@@ -118,15 +118,10 @@ func TestChainRPC(t *testing.T) {
 	blockNumber, err = common.HexToUint(header.Number)
 	require.NoError(t, err)
 	assert.GreaterOrEqual(t, blockNumber, uint(1))
-
-	var blockHash string
-	fetchWithTimeout(ctx, t, "chain_getBlockHash", "[]", &blockHash)
-	assert.Regexp(t, regex32BytesHex, blockHash)
-	assert.NotEqual(t, finalizedHead, blockHash)
```
Comment (qdm12) [inline tests/rpc/rpc_03-chain_test.go:125]: Why is this removed?

**Review by jimjbrettj — APPROVED** [review]
great work, besides what quentin already commented on this looks good to me

File: tests/rpc/rpc_01-system_test.go (hunk 2 above)
Comment (EclesioMeloJunior) [reply tests/rpc/rpc_01-system_test.go:83]: I am investigating why the `healthResponse.IsSyncing` is always true when we use `westend-dev` but when we use `gssmr` the `healthResponse.IsSyncing` is false

File: lib/babe/epoch_integration_test.go [hunk 5]
```diff
@@ -180,7 +180,7 @@ func TestService_getLatestEpochData_epochData(t *testing.T) {
 	require.NoError(t, err)
 
 	data := &types.EpochData{
-		Authorities: auths[:3],
```
Comment (timwu20) [inline lib/babe/epoch_integration_test.go:183]: why were we doing this before?

Comment (EclesioMeloJunior) [reply lib/babe/epoch_integration_test.go:183]: Because in the `gssmr` spec file there are 9 authorities and this was to use only 3 out of the 9 in the spec file

File: tests/rpc/rpc_01-system_test.go (hunk 2 above)
Comment (EclesioMeloJunior) [reply tests/rpc/rpc_01-system_test.go]: I fixed this by using `westend-local` instead of `westend-dev`

File: tests/rpc/rpc_03-chain_test.go [hunk 6]
```diff
@@ -103,11 +103,11 @@ func TestChainRPC(t *testing.T) {
 		assert.Regexp(t, regexBytesHex, digestLog)
 	}
 	block.Block.Header.Digest.Logs = nil
-	assert.Len(t, block.Block.Body, 1)
+	assert.NotNil(t, len(block.Block.Body), 1)
```
Comment (qdm12) [inline tests/rpc/rpc_03-chain_test.go:106]:
```suggestion
	assert.NotEmpty(t, block.Block.Body)
```

File: lib/utils/utils.go (hunk 1 above)
Comment (EclesioMeloJunior) [reply lib/utils/utils.go]: Removed the delta

File: tests/rpc/rpc_03-chain_test.go (hunk 4 above)
Comment (EclesioMeloJunior) [reply tests/rpc/rpc_03-chain_test.go]: Put it back, I removed for a moment and I forget to rollback

File: tests/rpc/rpc_03-chain_test.go (hunk 6 above)
Comment (EclesioMeloJunior) [reply tests/rpc/rpc_03-chain_test.go:106]: Sorry, fixed!

**Review by kishansagathiya — APPROVED** [review]

File: lib/utils/utils.go [hunk 7]
```diff
@@ -148,41 +148,14 @@ func KeystoreFilepaths(basepath string) ([]string, error) {
...
-// and fails the test if it cannot find it.
-func GetGssmrGenesisRawPathTest(t *testing.T) string {
-	t.Helper()
-	path, err := GetGssmrGenesisRawPath()
-	require.NoError(t, err)
-	return path
-}
-
-// GetGssmrGenesisRawPath gets the gssmr raw genesis path
+// GetGssmrGenesisRawPath gets the gssmr genesis path
```
Comment (qdm12) [inline lib/utils/utils.go:151]: supernit can't we keep it to how it was?

```suggestion
// GetGssmrGenesisRawPath gets the gssmr raw genesis path
```

**Review by qdm12 — APPROVED** [review]
Awesome 🎉

**Review by timwu20 — APPROVED** [review]
